OpenRA.Game/GameRules/WeaponInfo.cs

```diff
@@ -39,9 +39,12 @@ public sealed class WeaponInfo
 		[Desc("The maximum range the weapon can fire.")]
 		public readonly WDist Range = WDist.Zero;
 
-		[Desc("The sound played when the weapon is fired.")]
+		[Desc("The sound played each time the weapon is fired.")]
 		public readonly string[] Report = null;
 
+		[Desc("Sound played only on first shot.")]
+		public readonly string[] FirstBurstReport = null;
```

Comment:
Pedantic nit: this suffers from the same ambiguity as `BustReport`, where someone may assume that this sound plays only once ever.  How about `BurstStartReport` or `StartBurstReport`?